Sources/Commands/SwiftTool.swift

```diff
@@ -433,22 +433,33 @@ public class SwiftTool<Options: ToolOptions> {
         guard result.exitStatus == .terminated(code: 0) else {
             throw ProcessResult.Error.nonZeroExit(result)
         }
+
+        // Create backwards-compatibilty symlink to old build path.
+        let oldBuildPath = buildPath.appending(component: options.configuration.dirname)
+        if exists(oldBuildPath) {
+            try removeFileTree(oldBuildPath)
+        }
+        try createSymlink(oldBuildPath, pointingAt: plan.buildParameters.buildPath, relative: true)
```

Comment:
By the way, did you test your patch w/o this? I wonder if we are relying on it for tests to pass? Not necessarily a good thing, but probably a good thing to know.